### DISCUSSION TITLE
Hindi translations

### DISCUSSION THREAD

**Discussion by prakhar599** [fastapi/fastapi#10573] [Translations]
Unlock the Power of Hindi!

Imagine turning documentation translation into a thrilling journey. We propose an exciting approach to boost community participation in translating FastAPI docs into Hindi. The concept is simple but fun.

1. **Select a Page:** Indicate which documentation page you'd like to translate to Hindi, ensuring efficient coordination among contributors.

2. **Create a PR:** Initiate a Pull Request (PR) for your translation and be sure to tag this issue within the PR for clear tracking.

3. **Review and Refine:** Address potential issues and feedback raised during the review process to enhance the quality of your translation.

4. **Merging Success:** Once the translation meets the necessary standards, it will be merged into the FastAPI documentation, benefiting the Hindi-speaking audience.

**Comment by raches** [comment]
मैं हिंदी अनुवाद में योगदान देना चाहता हूं।

**Comment by YuriiMotov** [comment]
The process has changed, now we do translations automatically with AI.

We are currently working on adding Hindi translation. Here is the related discussion: https://github.com/fastapi/fastapi/discussions/15511
